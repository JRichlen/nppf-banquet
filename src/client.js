import React, {Fragment} from 'react';
import {Header} from './components/header';

export const Client = () => (
    <Fragment>
        <Header/>
    </Fragment>
);
