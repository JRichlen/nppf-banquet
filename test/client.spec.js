import React from "react";
import renderer from "react-test-renderer";
import { Client } from "../src/client";

describe("Client", () => {
  test("should render", () => {
    // when
    const component = renderer.create(<Client />);

    // then
    expect(component.toJSON()).toMatchSnapshot();
  });
});
