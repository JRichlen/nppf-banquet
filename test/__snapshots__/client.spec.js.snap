// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Client should render 1`] = `
<Fragment>
  <Header />
</Fragment>
`;
